daemon/stats/collector_unix.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/opencontainers/runc/libcontainer/system"
 )
 
+/*
```

Comment:
It is sadly not possible to do this in go only, `sysconf` isn't exposed. I discussed some other alternative options in https://github.com/docker/docker/pull/31579#issuecomment-284477640 but they are all IMHO worse.

I've no opinion on `pkg` vs not, but perhaps we could leave that refactoring until after 17.04?